Package

```diff
@@ -19,7 +19,8 @@ case "$arch" in
 esac
 case "$package_manager" in
         deb)
-                ;;
+            ;;
```

Comment:
Why no curl dependencies for a Debian package?

Reply:
As it was not required in case of debian. Well tested with the changes